packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -119,14 +149,27 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
     updateSingle('currentDose', currentDose);
   };
 
+  if (isLoading || vaccineState === undefined)
```

Comment:
Always have a `{` block after an `if`.

```suggestion
  if (isLoading || vaccineState === undefined) {
```